pkg/cri/resource-manager/flags.go

```diff
@@ -58,6 +59,8 @@ func init() {
 		"Fallback configuration to use unless/until one is available from the cache or agent.")
 	flag.StringVar(&opt.ForceConfig, "force-config", "",
 		"Configuration used to override the one stored in the cache. Does not override the agent.")
+	flag.StringVar(&opt.ForceConfigSignal, "force-config-signal", "SIGHUP",
```

Comment:
Why would anyone use anything other than `SIGHUP`? I think this config option might not be needed.

Reply:
SIGUSR might be used, too

Reply:
But, I'm not sure if this really needs to be configurable

Reply:
> Why would anyone use anything other than `SIGHUP`? I think this config option might not be needed.

I fully agree with your question. But I can't agree with your conclusion.

*Trust is good, control is better.* I would trust you or Markus any day to not do utterly stupid things. But we pull in an increasing number of packages written by all kinds of folks. We don't know them, we can't trust them. Hence we need control. The only thing we can really control in is our own code base.

Plus, with this being configurable you get for free the possibility to disable it.